src/renderer/rendered_window.rs

```diff
@@ -367,6 +367,7 @@ impl RenderedWindow {
         let transparent_floating = self.anchor_info.is_some() && has_transparency;
 
         if self.anchor_info.is_some()
+            && self.anchor_info.as_ref().unwrap().sort_order != settings.no_multigrid_zindex
```

Comment:
Might want to combine these two? As in e.g.:

```suggestion
        if self.anchor_info
            .as_ref()
            .map_or(false, |info| info.sort_order != settings.no_multigrid_zindex)
```